data/net/minecraft/world/entity/ai/goal/Goal.mapping

```diff
@@ -1,13 +1,13 @@
 CLASS net/minecraft/world/entity/ai/goal/Goal
 	METHOD canContinueToUse ()Z
-		COMMENT Returns whether an in-progress EntityAIBase should continue executing
+		COMMENT @return whether the goal should continue executing
 	METHOD canUse ()Z
 		COMMENT Returns whether execution should begin. You can also read and cache any state necessary for execution in this method as well.
```

Comment:
```suggestion
		COMMENT @return whether execution should begin. You can also read and cache any state necessary for execution in this method as well.
```

Reply:
In this situation, had only the first sentence exist there, the suggestion to change from method description to using the `@return` tag would be fine. However, the second sentence alongside the first sentence makes this suggestion invalid since the second sentence should not be part of the `@return` tag, as it has no relation to the returned value.